bundler.d/krb5.rb

```diff
@@ -1,4 +1,4 @@
 group :krb5 do
-  gem 'rkerberos', '>= 0.1.1'
-  gem 'gssapi'
+  gem 'rkerberos', '>= 0.1.1', :platform => :ruby
+  gem 'gssapi', :platforms => :ruby
```

Comment:
I don't like limiting the platform to `ruby` here. While bundler won't complain during `bundler install` on Windows, the resulting installation isn't going to be functional. I prefer that users using bundler-based installs explicitly skipped groups, ie: `bundler install --without krb5`. I'll make sure the install docs get updated if this information isn't there.